### PR TITLE
ACID logo in dashboard and navbar

### PR DESCRIPTION
**Pull request by pawelzny**

### CODE REVIEW

File: dashboard/templates/base.html [hunk 1]
```diff
@@ -11,22 +20,25 @@
...
             <span class="navbar-toggler-icon"></span>
         </button>
         <div class="collapse navbar-collapse" id="navbarSupportedContent">
             <ul class="navbar-nav mr-auto">
-                <li class="nav-item active">
+                <li class="nav-item {% if request.path == "/status" %}active{% endif %}">
                     <a class="nav-link" href="{{ url_for('status.show_status') }}">Status</a>
                 </li>
-                <li class="nav-item">
+                <li class="nav-item {% if request.path == "/builds" %}active{% endif %}">
```
Comment (kam193) [inline dashboard/templates/base.html]: This statement is not correct - when path is e.g. /builds/2, link will be not active. Use e.g. ``{% if request.path.startswith("/status") %}``

File: dashboard/templates/base.html [hunk 2]
```diff
@@ -11,22 +20,25 @@
...
+        </a>
         <button class="navbar-toggler" type="button" data-toggle="collapse"
                 data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent"
                 aria-expanded="false" aria-label="Toggle navigation">
             <span class="navbar-toggler-icon"></span>
         </button>
         <div class="collapse navbar-collapse" id="navbarSupportedContent">
             <ul class="navbar-nav mr-auto">
-                <li class="nav-item active">
+                <li class="nav-item {% if request.path == "/status" %}active{% endif %}">
```
Comment (kam193) [inline dashboard/templates/base.html]: This statement is not correct - when path is e.g. /status/check, link will be not active. Use e.g. ``{% if request.path.startswith("/status") %}``

File: dashboard/templates/base.html [hunk 3]
```diff
@@ -2,7 +2,16 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>{% block title %}Zuul Dashboard{% endblock %}</title>
+    <title>{% block title %}ACID for Zuul gating system{% endblock %}</title>
```
Comment (kawazaki0) [inline dashboard/templates/base.html]: Let me suggest, that it should be changed for each subpage. The title on a webpage is for example used in name of bookmark and it should stand for the name of service. I think that `Zuul gating system` has too much details. Maybe `Awesome CI Dashboard - {subpage}` or  `ACID - {subpage}`?

Comment (pawelzny) [reply dashboard/templates/base.html]: Good point. Totally agree.

**Pull request merged**